views/jobs/new.ejs

```diff
@@ -126,7 +126,7 @@
           <!-- 9. Do you offer any of these options? -->
           <div class="tab">
             <section class="questions">
-              <div class="sentence">What benefits or work-related extras do you offer?</div>
+              <div class="sentence">What about the benefits you offer and work culture of the company?</div>
```

Comment:
I wouldn't put the "What about the..." like this. I don't have a good alternative for now so maybe **let's discuss this on Wednesday with everyone**... also maybe showing some other options (right now we only show benefits, we could also show something like "regular feedback" and so on)

Reply:
yup let me know after all bc i wont be able to join on wednesday <3